### DISCUSSION TITLE
desktop apps

### DISCUSSION THREAD

**Discussion by wild-fruits** [bevyengine/bevy#13509] [Q&A]
I'm wondering if it's okay to build desktop applications using bevy

Because I love freedom and expression

**Comment by Luracasmus** [comment]
I would assume it's definitely possible, and probably not that hard either
[This example](https://bevyengine.org/examples/UI%20(User%20Interface)/ui/) uses `WinitSettings::desktop_app()` to optimize for saving power when there's no user input, since it's not a game, and draws some UI nodes to the screen

**Reply by wild-fruits** [reply]
Thank you sir for your answer,
This helped me a lot.

**Reply by Luracasmus** [reply]
> Thank you sir for your answer, This helped me a lot.

I'm glad I could help!
